tests/sentry/tasks/test_delete_seer_grouping_records.py

```diff
@@ -33,50 +32,43 @@ def test_delete_seer_grouping_records_by_hash_batches(
+    @patch(
+        "sentry.tasks.delete_seer_grouping_records.delete_seer_grouping_records_by_hash.apply_async"
+    )
+    def test_call_delete_seer_grouping_records_by_hash_simple(
+        self, mock_apply_async: MagicMock
+    ) -> None:
+        """
+        Test that call_delete_seer_grouping_records_by_hash correctly collects hashes
+        and calls the deletion task with the expected parameters.
+        """
         self.project.update_option("sentry:similarity_backfill_completed", int(time()))
 
-        group_ids, hashes = [], []
+        group_ids, expected_hashes = [], []
         for i in range(5):
             group = self.create_group(project=self.project)
             group_ids.append(group.id)
             group_hash = GroupHash.objects.create(
                 project=self.project, hash=str(i) * 32, group_id=group.id
             )
-            hashes.append(group_hash.hash)
+            expected_hashes.append(group_hash.hash)
+
         call_delete_seer_grouping_records_by_hash(group_ids)
-        mock_logger.info.assert_called_with(
-            "calling seer record deletion by hash",
-            extra={"project_id": self.project.id, "hashes": hashes},
-        )
 
-    @patch("sentry.tasks.delete_seer_grouping_records.delete_seer_grouping_records_by_hash")
-    @patch("sentry.tasks.delete_seer_grouping_records.logger")
+        # Verify that the task was called with the correct parameters
+        mock_apply_async.assert_called_once_with(args=[self.project.id, expected_hashes, 0])
+
+    @patch(
+        "sentry.tasks.delete_seer_grouping_records.delete_seer_grouping_records_by_hash.apply_async"
+    )
     def test_call_delete_seer_grouping_records_by_hash_no_hashes(
-        self, mock_logger, mock_delete_seer_grouping_records_by_hash
-    ):
+        self, mock_apply_async: MagicMock
+    ) -> None:
         self.project.update_option("sentry:similarity_backfill_completed", int(time()))
 
         group_ids = []
         for _ in range(5):
             group = self.create_group(project=self.project)
             group_ids.append(group.id)
         call_delete_seer_grouping_records_by_hash(group_ids)
-        mock_logger.info.assert_called_with(
-            "calling seer record deletion by hash",
-            extra={"project_id": self.project.id, "hashes": []},
-        )
-        mock_delete_seer_grouping_records_by_hash.assert_not_called()
-
-    @patch("sentry.tasks.delete_seer_grouping_records.logger")
-    def test_call_delete_seer_grouping_records_by_hash_no_group_ids(self, mock_logger):
-        call_delete_seer_grouping_records_by_hash([])
-        mock_logger.info.assert_not_called()
-
-    @patch("sentry.tasks.delete_seer_grouping_records.delete_project_grouping_records")
-    def test_call_delete_project_and_delete_grouping_records(
-        self, mock_delete_project_grouping_records
-    ):
-        call_seer_delete_project_grouping_records(self.project.id)
```

Comment:
This is the function call which is better tested here:
https://github.com/getsentry/sentry/blob/c20ff4ce3e92b2b9380a0f6aaa6424f06c6fa388/tests/sentry/api/endpoints/test_project_details.py#L1555-L1563